src/Servant/EDE.hs

```diff
@@ -93,16 +94,20 @@ import qualified Data.Vector         as V
 -- This would try to load @home.tpl@, printing any error or
 -- registering the compiled template in a global (but safe)
 -- compiled template store, if successfully compiled.
+type Filters = (Text,Term)
```

Comment:
@alpmestan  Thanks! You are right, it should be `Filter`. I also moved it up -- I have never put anything on Hackage before so I was just pattern matching what you had done and did not realize how the comments will render.